libr/asm/arch/avr/format.c

```diff
@@ -156,10 +156,34 @@ static int formatDisassembledOperand(avrDisassembleContext *context, char *strOp
 			fOptions.addressFieldWidth, context->longAddress);
 		break;
 	case OPERAND_IO_REGISTER:
-		retVal = sprintf(strOperand, "%s%02x",
-			OPERAND_PREFIX_IO_REGISTER,
-			dInstruction.operands[operandNum]);
+	{
+		char *current_register = NULL;
+		int is_register_found = 0;
+		switch(dInstruction.operands[operandNum])
+		{
+		    case 0x3f:
+		        current_register = "sreg\0";
```

Comment:
why did you resolved all my commit suggestions without applying them? this is wrong

Reply:
because I have not pushed yet. this suggestion has been solved locally. I am going to push a not finished work to show you.